public/projects.json

```diff
@@ -1,66 +1,35 @@
 [
   {
-    "name": "ndn-cxx",
-    "skip_job": true
```

Comment:
I just *assumed* this stuff is unused, a quick grep didn't give any hits, but please double check.